### DISCUSSION TITLE
FEP calculations: Are the MDP files templates?

### DISCUSSION THREAD

**Discussion by a-ws-m** [Becksteinlab/MDPOW#270] [Q&A] [answered]
Hi, I'm trying to put together an example using Martini 3 to demonstrate how `Forcefield`s work in this [PR](https://github.com/Becksteinlab/MDPOW/pull/267). I'm adapting some free energy calculation MDP files from the [Martini tutorial](http://www.cgmartini.nl/index.php/tutorials-general-introduction-gmx5/partitioning-techniques) on free energy techniques. I'm not sure exactly what to change in these files to make them work with the package.

In particular, from looking at the `bar_*.mdp` files in the repo, the `couple-moltype` is set to `SOLUTE`, which I don't think is a valid Gromacs group, but on the other hand I can't find where it is in the code that this line, or any others, are modified.

Which, if any, of the lines in the MDP files should be mutable?

**Comment by a-ws-m** [comment]
@orbeckst just wanted to ping you about this.

**Reply by orbeckst** [reply]
Sorry for the delay. I’m traveling and I don’t always see what’s happening in Discussions. Pinging me is a good approach!

**Comment by orbeckst** [comment, marked as answer]
Technically speaking all lines are mutable because of the “sed” approach that we used to changing these templates. Originally the idea was that each file should be “runnable” but as this example shows, you still need the appropriate NDX file to define a group. 

However, going forward it would be a good idea to use more of an explicit and modern templating approach (as you have used in your PR).
